src/formpack/utils/xls_to_ss_structure.py

```diff
@@ -162,34 +161,35 @@ def xlsx_value_to_str(value):
             return str(value).replace(chr(160), ' ')
 
     def xlsx_to_dict_normal_sheet(sheet):
-
-        # Check for duplicate column headers
         column_header_list = list()
-        for cell in sheet[1]:
-            column_header = cell.value
-            # xls file with 3 columns mostly have a 3 more columns that are
-            # blank by default or something, skip during check
```

Comment:
Removed comment because none of the code did anything in particular with 3 columns